src/backend/vulkan/src/lib.rs

```diff
@@ -732,9 +734,9 @@ impl adapter::PhysicalDevice<Backend> for PhysicalDevice {
                         | Features::STORAGE_TEXTURE_DESCRIPTOR_INDEXING
                         | Features::UNSIZED_DESCRIPTOR_ARRAY
                 ) {
-                    Some(*EXT_DESCRIPTOR_INDEXING)
+                    vec![*KHR_MAINTENANCE3, *EXT_DESCRIPTOR_INDEXING]
```

Comment:
what about the place where we *expose* the features for descriptor indexing? These should also check for `KHR_MAINTENANCE3`

Reply:
Discussed on matrix; unneeded